lib/viewvc.py

```diff
@@ -5299,11 +5329,18 @@ def view_query(request):
         generate_page(request, "query_results", data)
 
 
+def view_unspecified():
+    # This is just a placeholder for the default view of a file or directory.  It's
+    # never called directly, but is used as a special value for the "view_func".
+    pass
```

Comment:
This can be called by manually specifying `view=file` in query string because of  specification in `_views` dict,  and if it is called,  it throws Python exception:
```
TypeError: view_unspecified() takes 0 positional arguments but 1 was given
```
How about add `request` parameter to this function?  And then it could throw a ViewVCException for "Bad Request".